lib/cdo/rack/attack.rb

```diff
@@ -57,4 +58,23 @@ def write?
       req.write? && %r{^/v3/shared-properties/([^/]+)/[^/]+$}.match(req.path).try(:[], 1)
     end
   end
+
+  ActiveSupport::Notifications.subscribe('rack.attack') do |_, _, _, _, req|
+    if CDO.newrelic_logging
```

Comment:
We still need to actually turn this on somewhere, right? Do we prefer CDO.* for these sorts of things vs DCDO?